modules/tensor_mechanics/doc/content/source/kernels/ADDynamicStressDivergenceTensors.md

```diff
@@ -0,0 +1,13 @@
+# ADDynamicStressDivergenceTensors
+
+!syntax description /Kernels/ADDynamicStressDivergenceTensors
+
+## Description
+
+This class computes the stress divergence and the stiffness proportional Rayleigh damping for structural dynamics problems. Each ADDynamicStressDivergenceTensors input block computes force in one direction. So, a separate ADDynamicStressDivergenceTensors input block should be set up for each coordinate direction. The [DynamicTensorMechanics](/DynamicTensorMechanicsAction.md) action automatically sets up the ADDynamicStressDivergenceTensors input block in all coordinate direction. More information about the residual calculation and usage can be found at [Dynamics](Dynamics.md).
```

Comment:
```suggestion
This class computes the stress divergence and the stiffness proportional Rayleigh damping for structural dynamics problems. Each ADDynamicStressDivergenceTensors input block computes force in one direction. So, a separate ADDynamicStressDivergenceTensors input block should be set up for each coordinate direction. The [DynamicTensorMechanics](/DynamicTensorMechanicsAction.md) action automatically sets up the ADDynamicStressDivergenceTensors input block in all coordinate directions. More information about the residual calculation and usage can be found at [Dynamics](Dynamics.md).
```

Reply:
P.S.: This is one reason I prefer hard wrapped .md files (and .tex files). Spot the change!

Reply:
What do you mean by hard wrapped? An enforced line length?

Reply:
Sorry that was a typo, I meant "hard rapped".

Reply:
Yeah, of course, hard line breaks before the line length exceeds 80 chars. Most editors have a reflow hotkey, don't they?

Reply:
> Sorry that was a typo, I meant "hard rapped".

You like hard rap? 😄 

> line length exceeds 80 chars

80? Is that a standard of some kind? I personally use 100 for consistency with our clang-format, and yes I set that with my editor.

Reply:
https://hackernoon.com/does-column-width-of-80-make-sense-in-2018-50c161fbdcf6

80 chars is an ancient convention! And you can debate (see above) whether it makes sense for code with wider screens. But I think the wider screens argument is misguided. Just use multiple editor views side by side. Text with longer lines is simply less readable, regardless of screen width:

https://practicaltypography.com/line-length.html